test/CSSTransitionGroup-test.js

```diff
@@ -298,7 +298,9 @@ describe('CSSTransitionGroup', () => {
       state = { showChild: true };
 
       componentDidMount() {
-        this.setState({ showChild: false });
+        setTimeout(() => {
+          this.setState({ showChild: false });
+        });
```

Comment:
This diff is for to fix `react/no-did-mount-set-state`...
I guess there are better ways...